go.mod

```diff
@@ -2,6 +2,8 @@ module github.com/ONSdigital/dp-identity-api
 
 go 1.21
 
+replace google.golang.org/protobuf => google.golang.org/protobuf v1.33.0
```

Comment:
It's good practice to only upgrade the specific version that we're calling to ensure that we then don't downgrade by accident later.

Reply:
Odd this hasn't shown up on the go.sum though.